components/automate-chef-io/content/docs/api-tokens.md

```diff
@@ -10,8 +10,12 @@ toc = true
     weight = 70
 +++
 
+{{< info >}}
+This content is specific to IAM v1, see [IAM v2 Overview]({{< ref "iam-v2-overview.md#" >}}) for IAM v2 specific information.
+{{< /info >}}
+
 Chef Automate has two different types of API tokens: administrative and standard.
-With an administrative token you can access the entire Chef Automate API--including administrative tasks such as [managing local users]({{< relref "users.md" >}}) and [managing local teams]({{< relref "teams.md" >}}) and [managing authorization policies]({{< relref "authorization-overview.md" >}}).
+With an administrative token you can access the entire Chef Automate API--including administrative tasks such as [managing local users]({{< relref "users.md" >}}) and [managing local teams]({{< relref "teams.md" >}}) and [IAM v1 overview]({{< relref "iam-v1-overview.md" >}}).
```

Comment:
this last one is a little inconsistent with the `managing foo` structure . . . maybe

`[IAM v1 overview]` -> `[general authorization]` or `[refining IAM permissions]` or something

also, i know this was here before, but I think we should replace the first `and` with a comma. instead of having two `ands`